app-edge-complete.template.json

```diff
@@ -34,10 +34,6 @@
       "name": "edge-sip2",
       "version": "latest"
     },
-    {
-      "name": "edge-fqm",
-      "version": "latest"
-    },
```

Comment:
please also delete edge-orders and thinking that we need to increase major version in pom file since it is breaking change